sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts

```diff
@@ -220,11 +220,11 @@ describe("Certificates client - create, read, update and delete", () => {
   });
 
   it("can get a certificate's secret in PKCS 12 format", async function(this: Context) {
-    recorder.skip("browser", "This test uses the file system.");
-    // Skipping this test from the live browser test runs, because we use the file system.
-    if (!isNode) {
```

Comment:
I think we still need this unfortunately as it would not work at all in browsers (not even in live mode) due to the `fs` dependency...

Reply:
That's right!